Practic/Practic/Controllers/Data.cs

```diff
@@ -17,30 +14,50 @@ public DataController(IConfiguration config)
         {
             _configure = config;
             _name = _configure.GetValue<string>("DataFile:Data");
-
         }
 
         [HttpGet("DataFile")]
         public string GetJsonData()
         {
             return _name;
-
         }
 
         [HttpGet("TableData")]
         public string GetTable()
         {
-            return DataTable.GetTextFromFile(_name);
-
+            return DataTable.JsonDataFileName(_name);
         }
 
-        [HttpGet("TableData_Json")]
+        [HttpGet("TableData-Json")]
         public List<DataTable> GetJson()
         {
-            return DataTable.CreateTableWithOutSerialize(_name);
+            return DataTable.GetDeserialisedTableFromFile(_name);
+        }
+
+        [HttpGet("Item")]
+        public Item GiveAll()
+        {
+            var a = new Item();
+            return a;
+        }
 
+        [HttpGet("Item{Brand}")]
+        public Item GiveAll2()
+        {
+            var a = new Item();
+            return a;
         }
 
+        [HttpPost("Item")]
+        public Item PutAll()
+        {
+            var b = new Item();
+            
+            b.Post();
+
+            return b;
+
+        }
```

Comment:
this is not related to HW with deadline 03.11
Create new project and new PR for CRUD